biogrid/libs/grid-simulator/src/lib/interfaces.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { interfaces } from './interfaces';
 
 describe('interfaces', () => {
-  it('should work', () => {
+  test('should work', () => {
```

Comment:
can we be more descriptive than "should work"

Reply:
Sure, all of these are auto generated by NX. I will make sure to keep a look out in future PRs!